snmp/datadog_checks/snmp/data/profiles/_juniper-dcu.yaml

```diff
@@ -18,7 +18,7 @@ metrics:
         OID: 1.3.6.1.2.1.31.1.1.1.1
         name: ifName
       table: ifXTable
-      tag: interface_name
+      tag: interface
```

Comment:
general thought, this is something we might want to consider when having multiple tags across multiple profiles: using the same tag name for same fields. For example, `ifName` should always be `interface`.
Without this, filtering can be harder

Reply:
already fixed while you were commenting :-)